client/components/Nav.tsx

```diff
@@ -1,16 +1,167 @@
+                    sx={{
+                        display: "flex", 
+                        justifyContent: "space-between",
+                        flexDirection: "row",
+                        flexWrap: "nowrap"
+                    }}
+                >
+                    <IconButton
+                        size="large"
+                        edge="start"
+                        color="inherit"
+                        aria-label="menu"
+                        sx={{ mr: 2 }}
+                        aria-controls="menu-appbar"
+                        aria-haspopup="true"
+                        onClick={e => setMenuIconAnchorEl(e.currentTarget)}
+                    >
+                        <MenuIcon />                
+                    </IconButton>
+                    <Menu
+                        anchorEl={menuIconAnchorEl}
+                        anchorOrigin={{
+                            vertical: 'top',
+                            horizontal: 'right',
+                        }}
+                        keepMounted
+                        transformOrigin={{
+                            vertical: 'top',
+                            horizontal: 'right',
+                        }}
+                        open={Boolean(menuIconAnchorEl)}
+                        onClose={e => setMenuIconAnchorEl(null)}
+                    >
+                        <MenuItem 
+                            onClick={e => {
+                                setMenuIconAnchorEl(null);
+                                navigate("/");
+                            }}
+                        > 
+                            Home
+                        </MenuItem>
+                        {!isLoggedIn && (
+                            <>
+                                <MenuItem 
+                                    onClick={e => {
+                                        setMenuIconAnchorEl(null);
+                                        navigate("/login");
+                                    }}
+                                > 
+                                    Login
+                                </MenuItem>
+                                <MenuItem 
+                                    onClick={e => {
+                                        setMenuIconAnchorEl(null);
+                                        navigate("/register");
+                                    }}
+                                > 
+                                    Register
+                                </MenuItem>
+                            </> 
+                        )}
+                    </Menu>
+                    <Box
+                        component="div"
+                        sx={{
+                            display: "flex",
+                            justifyContent: "center"
+                        }}
+                    >
+                        <Typography 
+                            variant="h6" 
```

Comment:
I think you need to set this as an h1 - it is an h1 because it represents the top-level header on this app. You can set custom styles to change the font size if you want, but I don't recommend setting it as an h6, it will lead to a poor experience for users on screen readers